### PR TITLE
Improve logging for `create_release?` logic

### PR DESCRIPTION
**Pull request by disposedtrolley**
This PR changes the log message generated after `create_release?` is called to determine if a release should be created for the current request. The previous log message didn't mention additional checks for `service_type` and `service_name` which factored into the outcome, and caused some confusion as to why releases were being skipped.

Since this log is returned in the response body from inbound webhooks, it'd be good to include this extra context.

### References
- Jira link: 

### Risks
- Low

### CODE REVIEW

File: app/controllers/integrations/base_controller.rb [hunk 1]
```diff
@@ -16,7 +16,8 @@ def create
 
     if branch
       create_release = project.create_release?(branch, service_type, service_name)
-      record_log :info, "Branch #{branch} is release branch: #{create_release}"
+      record_log :info, "Create release for "\
+        "branch [#{branch}], service_type [#{service_type}], service_name [#{service_name}]: #{create_release}"
```
Comment (grosser) [inline app/controllers/integrations/base_controller.rb:20]:
```suggestion
      record_log(
        :info, 
        "Created release for branch: #{branch}, service_type: #{service_type}, service_name: #{service_name}: #{create_release}"
      )
```

Comment (disposedtrolley) [reply app/controllers/integrations/base_controller.rb]: It's still a _little_ too long for Rubocop's liking :(

Comment (grosser) [reply app/controllers/integrations/base_controller.rb:20]: dammit :D

**Comment by disposedtrolley** [comment]
Do you mind merging this one in @grosser? I don't seem to have permission!

**Pull request merged**